python/examples/myProtocol.py

```diff
@@ -70,10 +74,9 @@ def postBinary(self, data):
     def startStream(self, topic):
         print("start", topic)
         # set up repeated send of images until unsubscribed.
-        # http://twistedmatrix.com/documents/current/core/howto/time.html
         if not self.loopTask:
-            self.loopTask = task.LoopingCall(self.pushImage)
-            self.loopTask.start(2.0)
+            self.loopTask = schedule_callback(0, self.pushImage)
```

Comment:
did you say that you need the `interval` vs just `timeout==schedule_callback` ?

Reply:
Yes, we are missing the equivalent of `twisted.server.task.loop_task` which takes an interval and a callback.  I have a prototype almost ready for the callback version, and the coroutine version wouldn't be too much different I don't think.  But I thought those could be added later too, since we can use the schedule methods to achieve a similar thing.  Let me know what you think though.

Reply:
we can add it later since it is only an example but you can not forget about it... (Maybe make an issue on remaining things)